app-backend/api/src/main/scala/annotation-project/AnnotationProjectPermissionRoutes.scala

```diff
@@ -97,11 +122,23 @@ trait AnnotationProjectPermissionRoutes
             })
           }).transact(xa).unsafeToFuture()
         } {
+          val distinctUserIds = acrList
```

Comment:
Found an interesting way to get the same thing from l&t lately.
```scala
val distinctUserIds = acrList.foldMap(acr => acr.subjectId.map(Set(_)).getOrElse(Set())).toList
```

Reply:
Ha yeah that's a much cleaner expression, I'll switch it :sunglasses: